cmake/modules/BuildJaeger.cmake

```diff
@@ -62,3 +83,13 @@ function(build_jaeger)
     BUILD_BYPRODUCTS ${CMAKE_BINARY_DIR}/external/lib/libjaegertracing.so
     )
 endfunction()
+
+build_jaeger()
+set_library_properties_for_external_project(opentracing::libopentracing
+  opentracing)
+set_library_properties_for_external_project(jaegertracing::libjaegertracing
+jaegertracing)
+if(NOT thrift_FOUND)
+set_library_properties_for_external_project(thrift::libthrift thrift)
+set(jaeger_base ${jaeger_base} thrift::libthrift PARENT_SCOPE)
```

Comment:
indent.

Reply:
@ideepika this comment has not been addressed. please add two spaces for indent.

Reply:
aah, thought it was resolved in followup commits as was marked outdated, sorry, updated

Reply:
could you please fix in the commit introducing the change?